arch/arm/core/aarch64/Kconfig

```diff
@@ -67,6 +67,9 @@ config AARCH64_IMAGE_HEADER
 config PRIVILEGED_STACK_SIZE
 	default 4096
 
+config KOBJECT_TEXT_AREA
+	default 512
```

Comment:
If this is simply for test and not needed in general, might be a good idea to make it conditional (i.e. `if TEST`)

Reply:
ack and fixed.